src/views/Completed.vue

```diff
@@ -3,192 +3,212 @@
-              Next Day Shipping
-              <p>{{ $t("Ordered") }} 28th January 2020 2:32 PM EST</p>
+              {{ shipmentMethod.val }}
+              <p>{{ shipmentMethod.groups }} {{ shipmentMethod.groups > 1 ? $t('orders') : $t('order') }}, {{ shipmentMethod.itemCount }} {{ shipmentMethod.itemCount > 1 ? $t('items') : $t('item') }}</p>
             </ion-label>
-          </div>
+          </ion-item>
         </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail slot="start">
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
+        <ion-button expand="block" class="bulk-action desktop-only" fill="outline" @click="shipOrderAlert">{{ $t("Ship") }}</ion-button>
+
+        <ion-card v-for="(orders, index) in completedOrders.list" :key="index">
+          <div class="card-header">
+            <div class="order-primary-info">
               <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
+                {{ orders.doclist.docs[0].customerName }}
+                <p>{{ $t("Ordered") }} {{ formatUtcDate(orders.doclist.docs[0].orderDate, 'dd MMMM yyyy t a ZZZZ') }}</p>
               </ion-label>
-            </ion-item>
-          </div>
+            </div>
 
-          <div class="product-metadata mobile-only">
-            <ion-note>49 {{ $t("pieces in stock") }}</ion-note>
-          </div>
-        </div>
+            <div class="order-tags">
+              <ion-chip outline>
+                <ion-icon :icon="pricetagOutline" />
+                <ion-label>{{ orders.doclist.docs[0].orderId }}</ion-label>
+              </ion-chip>
+            </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail slot="start">
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
+            <div class="order-metadata">
               <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
+                {{ orders.doclist.docs[0].shipmentMethodTypeDesc }}
+                <!-- TODO: add support to display the last brokered date, currently not getting
+                the date in API response -->
+                <!-- <p>{{ $t("Ordered") }} 28th January 2020 2:32 PM EST</p> -->
               </ion-label>
-            </ion-item>
+            </div>
           </div>
 
-          <div class="product-metadata mobile-only">
-            <ion-note>49 {{ $t("pieces in stock") }}</ion-note>
-          </div>
-        </div>
+          <div v-for="order in orders.doclist.docs" :key="order" class="order-item">
```

Comment:
Here orders is having order and order is having item. Taken care in next PR

Reply:
https://github.com/hotwax/fulfillment-pwa/pull/98/files#diff-4fd9ea4fa1ac3bf01bf1a44d1a3e45717e8108f5fb8549ae83a281680486c966R82